.github/workflows/main.yml

```
@@ -164,6 +164,7 @@ jobs:
           pip3 install --user /tmp/cvat_sdk/
           pip3 install --user -r tests/python/requirements.txt
           pytest tests/python/rest_api -k 'GET' -s
+          pytest tests/python/sdk
```

```suggestion
          pytest tests/python/sdk tests/python/cli
```

Probably, there is no reason to keep them separated. Though, almost all the functionality should be covered by SDK tests. This change requires `pip install cvat-cli/`